### PR TITLE
*STILL TESTING - DO NOT MERGE* Adds Vibrancy Trait

### PR DESCRIPTION
**Pull request by Jujumatic**
Added a new trait: Vibrancy

This is supposed to be like the opposite of the Monochromacy trait - it will make you see colours far more brightly and vibrantly than normal. It'll be a neutral trait cause fuck needing points to get traits lmao. Also it will be overridden by Monochromacy if you pick both at the same time because you can't see brighter colours if you can't see colours

Why - cause extra colour is cool as heck yo

### CODE REVIEW

**Comment by JohnGinnane** [comment]
don't forget to add pics

File: hippiestation/code/datums/traits/neutral.dm [hunk 1]
```diff
@@ -16,6 +16,24 @@
 /datum/trait/monochromatic/remove()
 	trait_holder.remove_client_colour(/datum/client_colour/monochrome)
 
+/datum/trait/vibrancy
+	name = "Vibrancy"
+	desc = "You must have taken some of the good stuff, because everything looks way brighter and far more vibrant!"
+	value = 0
+	lose_text = "<span class='notice'>It seems like everything is not so vibrant anymore...</span>"
+
+/datum/trait/vibrancy/add() = /datum/client_colour
```
Comment (ghost) [inline hippiestation/code/datums/traits/neutral.dm]: this isn't valid
make this:
`/datum/trait/vibrancy/add()`

File: hippiestation/code/datums/traits/neutral.dm [hunk 2]
```diff
@@ -16,6 +16,24 @@
...
+	if(trait_holder.client_colour == client_colour.monochrome)
+		to_chat(trait_holder, "<span class='warning'>You feel your vibrant vision having no effect because of your monochromacy!</span>")
+		sleep(20) //Don't want the two messages to pop up instantly, muh immershun
```
Comment (ghost) [inline hippiestation/code/datums/traits/neutral.dm]: firstly, the sleep shouldn't be used, and even if you were to use it, it should be before the to_chat() code

**Review by ghost — CHANGES_REQUESTED** [review]

File: hippiestation/code/datums/traits/neutral.dm (hunk 2 above)
Comment (Jujumatic) [reply hippiestation/code/datums/traits/neutral.dm]: After sleep it triggers the remove trait which activates the lose text. The to_chat is just to tell the person that having monochromacy and vibrancy at the same time won't work

**Comment by ghost** [comment]
Honestly a really cool idea/PR for someone who is just learning to code. A+

**Comment by Jujumatic** [comment]
Closing this pr because the vibrancy dream is dead (for now) opening a new pr as I have some other things instead

**Pull request closed without merging**